src/schema/objects/columns.yaml

```
@@ -48,6 +48,15 @@ cardiac:
   description: |
     continuous pulse measurement
   type: number
+channel:
+  name: channel
+  display_name: Channel
+  description: |
+    Channel(s) associated with an event.
+    If multiple channels are specified, separate with commas (`,`) or a
+    delimiter specified in the `"Delimiter"` field describing the `channel`
+    column.
```

```suggestion
    Channel(s) associated with an event.
    If multiple channels are specified, they MUST be separated with a delimiter
    (for example, a comma `,`) specified in the `"Delimiter"` field describing
    the `channel` column (in the corresponding `channels.json` file).
```

---

I'd rather use more sentences than multiple parenthetical clauses.

What do you think about:

```suggestion
    Channel(s) associated with an event.
    If multiple channels are specified, they MUST be separated by a delimiter
    specified in the `"Delimiter"` field describing the `channel` column.
    For example, channels separated with a comma (`,`) require the `events.json`
    file to contain `"channel": {"Delimiter": ","}`.
    In the absence of a delimiter, tools MUST interpret any character as being part
    of a channel name.
```

The final sentence is intended to prevent some tools from adopting an implicit convention that would make them incompatible with other tools.